SimplenoteUITests/Generic.swift

```diff
@@ -47,6 +47,23 @@ func toggleSwitchIfNeeded(_ switchElement: XCUIElement, _ value: String) {
     switchElement.tap()
 }
 
+func dismissVerifyEmailIfNeeded(using app: XCUIApplication) {
+    guard app.staticTexts["Review Your Account"].waitForExistence(timeout: 15) else { return }
+    app.buttons["icon cross"].tap()
+}
+
+func populateNoteList(title: String, numberToCreate: Int) {
+    for i in 1...numberToCreate {
+        let noteTitle = title + String(i)
```

Comment:
What do you think of adding a space here:

```suggestion
        let noteTitle = title + " " + String(i)
```

So that you will not have to include the space in the end of `title` argument?

Reply:
Yup can do that, I'll change it from a space to a `-` cos the title also needs to be called later in the test file `NoteList.longPressNote(title: noteTitle + "1")`

Having a `" 1"` would also look weird and as a bonus to changing it to a `-` means automatically testing special characters on note title.